nixos/modules/services/misc/matrix-synapse.nix

```
@@ -699,4 +699,11 @@ in {
       };
     };
   };
+
+  imports = [
+    (mkRemovedOptionModule ["services" "trusted_third_party_id_servers"] ''
```

Option path element missing, should be:
```suggestion
    (mkRemovedOptionModule [ "services" "matrix-synapse" "trusted_third_party_id_servers" ] ''
```